src/Libraries/CoreNodeModelsWpf/Controls/CustomSelectionControl.xaml

```diff
@@ -0,0 +1,109 @@
+<UserControl x:Class="CoreNodeModelsWpf.Controls.CustomSelectionControl"
```

Comment:
This is pretty different from before. Based on Michael's feedback about using existing resources, I revised this to use the +/- buttons we already have for the List Join component instead of the "Add" and "X" buttons. 🤷